src/Init/Prelude.lean

```diff
@@ -2169,7 +2169,7 @@ Folds a function over a list from the left:
 `foldl f z [a, b, c] = f (f (f z a) b) c`
 -/
 @[specialize]
-def List.foldl {α β} (f : α → β → α) : (init : α) → List β → α
+def List.foldl {α β : Type _} (f : α → β → α) : (init : α) → List β → α
```

Comment:
Could you please use `Type u` and `Type v` instead of `Type _`? We don't use `Type _` anywhere in the code base.